bootstrap/bootstrappers/testingbootstrap/testing_bootstrap.go

```diff
@@ -38,9 +38,9 @@ var bootstappers = []bootstrap.TestBootstrapper{
 	&invoice.Bootstrapper{},
 	&purchaseorder.Bootstrapper{},
 	&nft.Bootstrapper{},
-	&queue.Starter{},
 	p2p.Bootstrapper{},
 	documents.PostBootstrapper{},
+	&queue.Starter{},
```

Comment:
strange that previous code worked for integration tests.

Reply:
Why do you think so?
The starter bootstrapper only needs the queue server which is put into context by queueBoostrapper. I think it should be working before this change

Reply:
Queue must be started after all task types are registered, we were skipping the anchorTask before because start being before `documents.PostBootstrapper{}`

Reply:
I think that is still fine since the task registration can still happen after the queue starts